src/utils/Cookie.php

```diff
@@ -0,0 +1,60 @@
+<?php
+
+/*
+ * This file is part of RakLib.
+ * Copyright (C) 2014-2022 PocketMine Team <https://github.com/pmmp/RakLib>
+ *
+ * RakLib is not affiliated with Jenkins Software LLC nor RakNet.
+ *
+ * RakLib is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ */
+
+declare(strict_types=1);
+
+namespace raklib\utils;
+
+use raklib\utils\InternetAddress;
+use pocketmine\utils\Binary;
+use function random_int;
+use function crc32;
+
+final class Cookie{
```

Comment:
A cookie typically refers to a single piece of data rather than the entire store of cookies. Maybe rename this to something like `CookieStore`/`CookieCache`/`CookieManager`?